index.html

```diff
@@ -9222,6 +9222,240 @@ <h3 id="example-2">
+            This implements the pre-delay and the application of the reduction
+            gain.
+          </div>
+          <p>
+            The following algorithm describes the
+            <code>EnveloppeFollower</code> object to be applied to the input
+            signal to produce the gain reduction value. First, a couple
+            variables that persist accross invocation of this algorithm, tied
+            to this particular <code>EnveloppeFollower</code> instance.
+          </p>
+          <ul>
+            <li>Let <var>detector average</var> be a floating point number,
+            initialized to 1.0.
+            </li>
+            <li>Let <var>compression gain</var> be a foating point number,
+            initialized to 1.0.
+            </li>
+          </ul>
+          <p>
+            Now, the actual algorithm to determine the reduction gain, in
+            linear gain value.
+          </p>
+          <ol>
+            <li>Let <var>input signal</var> be the the block of audio to be
+            processed.
+            </li>
+            <li>Let <var>threshold</var>, <var>knee</var> have the value of the
+            <a>AudioParam</a> of the same name, <a>converted to linear unit</a>
+            sampled at the time of processing of this block (as <a>k-rate</a>
+            parameters).
+            </li>
+            <li>Let <var>ratio</var> have the value of the <code>ratio</code>
+            <a>Audioparam</a>, sampled at the time of processing of this block
+            (as a <a>k-rate</a> parameter).
+            </li>
+            <li>Let <var>attack</var> and <var>release</var> have the value of
+            the <a>AudioParam</a> of the same name, sampled at the time of
+            processing (those are <a>k-rate</a> parameters), converted to
+            sample-frames, based on the sample-rate of the
+            <a>BaseAudioContext</a> this <a>DynamicsCompressorNode</a> is
+            <a href="#associated">associated</a> with.
+            </li>
+            <li>Set <var>attack</var> and <var>release</var> to the
+            multiplication between themselves and the sample-rate for this <a>
+              BaseAudioContext</a>.
+            </li>
+            <li>For each samples <var>input</var> of <var>input signal</var>,
+            execute the following steps:
+              <ol>
+                <li>Let <var>envelope rate</var> be the result of computing the
+                <a href="envelope-rate">envelope rate</a>
+                </li>
+                <li>Let <var>attenuation</var> be the value of putting the
+                absolute value of <var>input</var> <a href="compression-curve">
+                  through the compression curve</a>.
+                </li>
+                <li>Add the difference between <var>attenuation</var> and <var>
+                  detector average</var>, multiplied by the <var>enveloppe
+                  rate</var> to <var>detector average</var>, and clamp
```

Comment:
I'll be spelling out this computation in mode details, so that the precedence is very clear.